pom.xml

```diff
@@ -238,16 +239,16 @@
     </pluginManagement>
 
     <plugins>
-      <plugin>
-        <artifactId>maven-toolchains-plugin</artifactId>
-        <executions>
-          <execution>
-            <goals>
-              <goal>toolchain</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
+<!--      <plugin>-->
+<!--        <artifactId>maven-toolchains-plugin</artifactId>-->
+<!--        <executions>-->
+<!--          <execution>-->
+<!--            <goals>-->
+<!--              <goal>toolchain</goal>-->
+<!--            </goals>-->
+<!--          </execution>-->
+<!--        </executions>-->
+<!--      </plugin>-->
```

Comment:
Toolchain cannot be used easily and anyway obsoleted by the fact that we control what's inside a docker image. So this would need to be moved in a profile that we can disable when running on GA